demo.py

```diff
@@ -58,7 +65,7 @@ def train_model(X_train, y_train, X_test, y_test, lmd):
     N_test = len(X_test)
 
     print("\n======================================")
```

Comment:
Actually I had not noticed this initially.  I started by removing it as you suggest, but then what I noticed is that it does actually serve a purpose in separating the output for the two datasets, if you run them both at the same time (which I hadn't been doing in my own testing).

For example, here is how that line separates the two, where I'm showing just the end of the output from the first data set, followed by the beginning of the next one:

```
QBoostPlus
accu (train):  0.97
accu (test):  0.89

============================
Method        Train  Test   
----------------------------
Adaboost      0.86   0.82   
DecisionTree  0.97   0.88   
Qboost        0.97   0.88   
QboostIt      0.97   0.89   
============================

======================================
Train: 379, Test: 190
Num weak classifiers: 35
Tree depth: 3
```

What would you think about just tweaking `demo.py` so that it only runs one data set at a time?  That would make more sense to me and IMO would be more "standard" behavior in terms of a command line interface.  That would obviate the need to worry about formatting for separating the two cases in the output.

Reply:
Agree. I figured that https://github.com/dwave-examples/qboost/pull/12#discussion_r564118702 would solve the issue of separating the two data sets instead, but this is more standard for our examples.

Reply:
Done, I have changed the command line syntax so that only one data set can be run at a time